proposals/0240-sdl-js-pwa.md

```diff
@@ -188,7 +188,7 @@ App HMI Type | `Application.category` |  _No_
 Additional App HMI Types | _No_ | `Application.additional_categories`
 App Name | `Application.display_names[0]` |  _No_
 App Icon | `Application.icon_url` |  _No_
-App Version | _No_ | `Application.package_version_string`
+App Version | _No_ | `Application.platform_version_string`
```

Comment:
Accidental deviation from the recommendation? Suggested change to `Application.platform_app_version` to remove unnecessary "_string" suffix and align with existing `platform_` attribute names.

Reply:
whoopsie...